### DISCUSSION TITLE
Does server have to be connected to internet

### DISCUSSION THREAD

**Discussion by bsjdsp** [apohl79/audiogridder#460] [Q&A]
This may be a silly question but can I make a offline server that I use that connects via Ethernet to use the application. Also can the computer that I use the plugins on also have wifi and audiogridder connected via Ethernet to stream plugins from a server. 

Basically i just want a offline server setup.

**Comment by CaBuffalo** [comment]
As long as you have all of your plugin loaded in your server, no, you don't
need to have it connected to the internet.

On Tue, Mar 30, 2021 at 11:52 PM bsjdsp ***@***.***> wrote:

> This may be a silly question but can I make a offline server that I use
> that connects via Ethernet to use the application. Also can the computer
> that I use the plugins on also have wifi and audiogridder connected via
> Ethernet to stream plugins from a server.
>
> Basically i just want a offline server setup.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/460>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6GNG5CA7Y75KY2QYXDTGLBBDANCNFSM42D3YGVQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by apohl79** [comment]
Yes you can. The server itself needs no internet connection. 

edit: thx @CaBuffalo for being faster :-)
